app/views/applications/new.html.erb

```diff
@@ -0,0 +1,14 @@
+<br>
+
+  <h1>Application Information</h1>
+<%= form_tag "/applications" do %>
+  <%= label_tag :user_name %><br>
+  <%= text_field_tag :user_name %><br>
+  <%= label_tag :address %><br>
+  <%= text_field_tag :address %><br>
+  <%= label_tag :description %><br>
+  <%= text_field_tag :description %><br>
+  <%= label_tag :pet_names %><br>
+  <%= text_field_tag :pet_names %><br>
+  <%= submit_tag "Submit" %>
```

Comment:
After reading the user story, it looks like the create application should only take a user_name when creating the application. So all of these other fields should not be a part of it (future user stories will include those fields). This means we'll need to change the requirements for an application creation to only validate a username.